src/common/GroupAvatar.js

```diff
@@ -1,11 +1,14 @@
 /* @flow strict-local */
 import React, { PureComponent } from 'react';
 import type { Node } from 'react';
-import { Text, View } from 'react-native';
+import { View } from 'react-native';
+import FontAwesome from 'react-native-vector-icons/FontAwesome';
 
+// $FlowFixMe[untyped-import]
+import Color from 'color';
```

Comment:
Nit: This should go above the blank line. The blank line is meant to separate third-party imports (of code that isn't ours) from imports of our own code.

Reply:
Sure, will do that 👍🏻